source/develop/release-management/features/storage/hotplugdisk.html.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Hot Plug and Unplug Disk
+category: template
+authors: derez, mkublin, ykaul
+wiki_category: Template
+wiki_title: Features/DetailedHotPlug
+wiki_revision_count: 25
+wiki_last_updated: 2014-07-13
```

Comment:
I may be wrong, but above wiki_* metadata are meaningless on new files, there's no corresponging wiki page on old wiki instance. @garrett ?